tests/cstor/cspc/provisioning/provisioning_negative_test.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2019 The OpenEBS Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provisioning
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"github.com/openebs/maya/pkg/debug"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+const HostIpPort = "146.148.72.182:30036"
```

Comment:
Can we use go flag here instead of defining a const and changing it .

Reply:
Having a flag for Service `NodePort`, and `IP` will be good.

Reply:
We will automate it via inside the BDD PR when it goes into pipeline -- to get the node IP.
Anyways -- via flag is also a good option.
Will do it a later PR too.